development.ini.example

```diff
@@ -216,9 +216,7 @@ stacks_enabled = False
 # updates.  Users have free-reign to override them for each kind of entity.  At
 # the end of the day, we only consider the requirements defined by single
 # updates themselves when gating in the backend masher process.
-site_requirements = depcheck upgradepath
-## Some day we'll have rpmgrill, and that will be cool.  Ask tflink.
-#site_requirements = depcheck upgradepath rpmgrill
+site_requirements = dist.rpmdeplint dist.upgradepath
```

Comment:
Please rebase this PR on the develop branch - once you've done that, please also make this setting commented by default (similar to most of the rest of the file) and then add this default to ```bodhi.server.config.BodhiConfig._defaults```. Thanks!